ci/bamboo/build-osx.sh

```diff
@@ -1,32 +1,37 @@
 #!/bin/bash
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
```

Comment:
The documentation for what to pass here is missing. I am guessing that the success of the OSX build depends on something higher up passing in the `--user` command-line option, but it's not documented in this script.

Practically, since this script is acting as an adapter for OSX nupic.bindings builds in our Bamboo CI, I would expect it to encapsulate anything specific to the OSX builds, such as passing of the `--user` command-line option to `build-and-test-nupic-bindings.sh` instead of expecting details such as this to be configured in Bamboo.